src/models/utils.js

```diff
@@ -1,9 +1,13 @@
+/* eslint-disable max-len */
 const MONTH_VALIDATION = /^([0]{1}[1-9]{1}|[1]{1}[0-2]{1})-[2]{1}[0]{1}[0-9]{2}$/;
 const PHONE_VALIDATION = /^[0]{1}[1-9]{1}[0-9]{8}$/;
 const SIRET_VALIDATION = /^\d{14}$/;
-
+const IBAN_VALIDATION = /FR\d{12}[0-9A-Z]{11}\d{2}$/;
```

Comment:
Ici, je ne sais pas si on ne devrait pas renseigner 2 lettres au debut plutot que FR, imaginons qu'on ai un RIB etranger, ca casserait ici. Tu en penses quoi ?

Reply:
à la base la regex c'était ça : 
^AL\d{10}[0-9A-Z]{16}$|^AD\d{10}[0-9A-Z]{12}$|^AT\d{18}$|^BH\d{2}[A-Z]{4}[0-9A-Z]{14}$|^BE\d{14}$|^BA\d{18}$|^BG\d{2}[A-Z]{4}\d{6}[0-9A-Z]{8}$|^HR\d{19}$|^CY\d{10}[0-9A-Z]{16}$|^CZ\d{22}$|^DK\d{16}$|^FO\d{16}$|^GL\d{16}$|^DO\d{2}[0-9A-Z]{4}\d{20}$|^EE\d{18}$|^FI\d{16}$|^FR\d{12}[0-9A-Z]{11}\d{2}$|^GE\d{2}[A-Z]{2}\d{16}$|^DE\d{20}$|^GI\d{2}[A-Z]{4}[0-9A-Z]{15}$|^GR\d{9}[0-9A-Z]{16}$|^HU\d{26}$|^IS\d{24}$|^IE\d{2}[A-Z]{4}\d{14}$|^IL\d{21}$|^IT\d{2}[A-Z]\d{10}[0-9A-Z]{12}$|^[A-Z]{2}\d{5}[0-9A-Z]{13}$|^KW\d{2}[A-Z]{4}22!$|^LV\d{2}[A-Z]{4}[0-9A-Z]{13}$|^LB\d{6}[0-9A-Z]{20}$|^LI\d{7}[0-9A-Z]{12}$|^LT\d{18}$|^LU\d{5}[0-9A-Z]{13}$|^MK\d{5}[0-9A-Z]{10}\d{2}$|^MT\d{2}[A-Z]{4}\d{5}[0-9A-Z]{18}$|^MR13\d{23}$|^MU\d{2}[A-Z]{4}\d{19}[A-Z]{3}$|^MC\d{12}[0-9A-Z]{11}\d{2}$|^ME\d{20}$|^NL\d{2}[A-Z]{4}\d{10}$|^NO\d{13}$|^PL\d{10}[0-9A-Z]{,16}n$|^PT\d{23}$|^RO\d{2}[A-Z]{4}[0-9A-Z]{16}$|^SM\d{2}[A-Z]\d{10}[0-9A-Z]{12}$|^SA\d{4}[0-9A-Z]{18}$|^RS\d{20}$|^SK\d{22}$|^SI\d{17}$|^ES\d{22}$|^SE\d{22}$|^CH\d{7}[0-9A-Z]{12}$|^TN59\d{20}$|^TR\d{7}[0-9A-Z]{17}$|^AE\d{21}$|^GB\d{2}[A-Z]{4}\d{14}$
et du coup je me suis dit que c'était un peu long pour rien. J'ai l'impression qu'il y a 2 cas où c'est plus que 2 lettres au début (MR13 et TN59). Après c'est peut-être des cas particulier qu'on aura jamais et on peut se permettre de mettre 2 lettres indifférenciées.

Reply:
Ah en effet c'est celle-ci est beaucoup trop longue ^^' 
A priori il n'y a pas de raison qu'on ai d'IBAN étranger donc je serais d'avis a laisser comme ca et si jamais on se rend compte que c'est trop contraignant, on pourra modifier la REGEX